oracle/Dockerfile

```diff
@@ -6,7 +6,7 @@
 
 # Pull base image
 ARG BASE_IMAGE_TAG
-FROM openjdk:${BASE_IMAGE_TAG:-11.0.10-jdk-oraclelinux8}
+FROM openjdk:${BASE_IMAGE_TAG:-16-jdk-oraclelinux8}
```

Comment:
Is this required? Maybe you can just add 16-jdk-oraclelinux8 to the build matrix? Preferable also in a different pr

Reply:
That's not required, you're right. Then the previous line with the installation of git wouldn't be required too.